ga4gh/backend.py

```diff
@@ -139,12 +168,9 @@ def _getContainer(self):
             raise exceptions.ReadGroupNotFoundException(readGroupId)
         return readGroup
 
-    def _getIterator(self):
-        iterator = self._container.getReadAlignments(
-            self._request.referenceName,
-            self._request.referenceId,
-            self._startPosition, self._request.end)
-        return iterator
+    def _search(self, start, end):
+        return self._container.getReadAlignments(
```

Comment:
whoops!  you almost re-introduced a bug here!  getReadAlignments has 4 arguments in this version, not 3